### ISSUE TITLE
WebSocket handshake: 'Upgrade' header must not appear more than once in a response

### ISSUE THREAD

**Issue by fyn-dev** [ktorio/ktor#256]
When I using websocket feature in ktor source example it works, but when I try to use in my project from maven gradle repository it doesn't work.

I getting error on client side of browser: **WebSocket handshake: 'Upgrade' header must not appear more than once in a response**

Here is my source code:

```
import io.ktor.application.install
import io.ktor.features.CallLogging
import io.ktor.features.DefaultHeaders
import io.ktor.routing.routing
import io.ktor.server.engine.embeddedServer
import io.ktor.server.jetty.Jetty
import io.ktor.websocket.Frame
import io.ktor.websocket.WebSockets
import io.ktor.websocket.readText
import io.ktor.websocket.webSocket
import kotlinx.coroutines.experimental.channels.consumeEach

fun main(args: Array<String>) {
    val server = embeddedServer(Jetty, 8080) {
        install(DefaultHeaders)
        install(CallLogging)
        install(WebSockets)

        routing {

            webSocket("/") {

                try {
                    incoming.consumeEach { frame ->
                        if (frame is Frame.Text)
                            send(Frame.Text("ECHO: ${frame.readText()}"))
                    }
                } finally {

                }
            }
        }
    }
    server.start(wait = true)
}
```

And here is my build.gradle file:

```
version '1.0-SNAPSHOT'

buildscript {
    ext.kotlin_version = '1.1.60'
    ext.ktor_version = '0.9.0'

    repositories {
        mavenCentral()
    }
    dependencies {
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }
}

apply plugin: 'kotlin'

repositories {
    mavenCentral()
    maven { url  "http://dl.bintray.com/kotlin/ktor" }
    maven { url "https://dl.bintray.com/kotlin/kotlinx" }
}

dependencies {
    compile "org.jetbrains.kotlin:kotlin-stdlib-jre8:$kotlin_version"
    compile "io.ktor:ktor-server-jetty:$ktor_version"
    compile "io.ktor:ktor-websockets:$ktor_version"
}

compileKotlin {
    kotlinOptions.jvmTarget = "1.8"
}
compileTestKotlin {
    kotlinOptions.jvmTarget = "1.8"
}
```

What's wrong? Why I having issue with duplicate 'Upgrade' header?

**Comment by cy6erGn0m** [comment]
Already fixed in 0.9.1-alpha-5

**Comment by fyn-dev** [comment]
@cy6erGn0m @orangy this issue only take place for Jetty, with Netty works fine
